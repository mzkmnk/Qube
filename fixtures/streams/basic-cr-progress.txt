# 基本CR進捗と確定ライン
Loading... 0%Loading... 25%Loading... 50%Loading... 75%Loading... 100%
Done
