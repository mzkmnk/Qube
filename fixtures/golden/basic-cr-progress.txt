Loading... 100%
Done
